docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  gollum:
+    image: "gollumwiki/gollum"
+    ports:
+      - "4567:4567"
+    volumes:
+      - "./wiki:/wiki"
```

Comment:
If I understand well this assumes the wiki will be in `.wiki/` rather than `./`. No strong feelings either way, but it should be documented. Also, just to make sure, this behavior will be specific to using Docker with the `compose` file, right? So if you just pull the `gollumwiki/gollum` image and `docker run` it, it will still expect the wiki to be in `./`?

Reply:
See my comment about testing. To test, it builds from the current directory and it wont pull from gollumwiki/gollum. My original commit was building. But then I got confused and I changed it to pull from gollumwiki/gollum. But that actually doesn't make sense.